### ISSUE TITLE
[BUG] @Builder on class level overrides  java.lang.Object.toString

### ISSUE THREAD

**Issue by nimo23** [rzwitserloot/lombok#2308]
**Describe the bug**
Using `@Builder` on class level overrides `java.lang.Object.toString`

**To Reproduce**
Use the following class in Eclipse IDE:

```
// eclipse ide shows message: overrides `java.lang.Object.toString` 
@Builder
public class User {
    private int id;
    private String name;
}
```

**Expected behavior**
No need to override `java.lang.Object.toString` for using builder pattern.

**Version info (please complete the following information):**
 - Lombok version: 1.18.10
 - Platform: Eclipse 2019-09 R (4.13.0)

**Comment by OLPMO** [comment]
Could you provide a minimal executable demo? @nimo23

**Comment by nimo23** [comment]
You can simply use this class in eclipse IDE with lombok integration:
```

import lombok.Builder;
import lombok.Getter;

// eclipse ide shows message: overrides `java.lang.Object.toString`
@Builder
@Getter
public class Item {
    private int id;
    private String name;
}
```

and eclipse shows overriding symbol on the `@Builder:`

![lombok-builder](https://user-images.githubusercontent.com/3045549/70696596-509f3400-1cc4-11ea-8204-8445e358d084.png)

**Comment by OLPMO** [comment]
Oh, thank you for your reply. I have made a minimal demo on my own.I try to decompile the class file of `User ` and found that  `User` does not override the `java.lang.Object.toString`.Or do you mean that it is just the error of tip of  lombok plugin in eclipse?@nimo23

**Comment by nimo23** [comment]
I dont know, I only see the green triangle left in my IDE beside the `@Builder`-Annotation which shows me the message: `overrides java.lang.Object.toString`.

**Comment by rspilker** [comment]
I can reproduce the behavior, but I don't think we can fix this.

For generated methods, we set the "position" at the annotation. The source position for generated setters will be the `@Setter`. This allows Eclipse to go to the Declaration if you press <kbd>F3</kbd>. 

Similar, if you use `@ToString` on a type, the line containing it will have the same green arrow:
```java
▲ @ToString
  public class User {
    private int id;
    private String name;
  }
```

Now `@Builder`. It generated a Builder class, and in that class it generates a `toString` method. The position of that method is that of the `@Builder annotation`.

You can verify in the following code by putting the caret in `toString` and pressing <kbd>F3</kbd>. You will see that eclipse will highlight `@builder`.
```java
▲ @Builder
  public class User {
      private int id;
      {
    	  builder().toString();
      }
  }
```

In short, this is expected behavior, and we don't plan to change this.
